lineage/Analyze.py

```diff
@@ -15,13 +15,41 @@ def submit(self, fn, *args, **kwargs):
 
 
 def Analyze(X, num_states, **kwargs):
-    """ Runs a tHMM and outputs the tHMM object, state assignments, and likelihood. """
+    """ 
+    Runs a tHMM and outputs the tHMM object, state assignments, and likelihood 
+    after fitting experimental data from :func: `Analyze_list`. 
+    
+    :param X: list of objects (cells) in a lineage
+    :type X: list
+    :param num_states: the number of states to identify
+    :type num_states: int
+    :return: the tHMM object after fitting
+    :rtype: object
```

Comment:
An `rtype` of object tells us nothing about what is returned. For example, here, this is a tHMM class, which is a specific type.

Reply:
It should _almost always_ be possible to say something more specific than object.